CONTRIBUTING.md

```diff
@@ -0,0 +1,154 @@
+# Contributing to insights-core-messaging
+
+## Introduction
+
+The insights-core-messaging project provides a framework for the
+building of insights archive processing applications.
+
+The framework is implemented in Python 3.7+.  The code management uses
+the [gitflow](https://nvie.com/posts/a-successful-git-branching-model/)
+branching strategy.  So normal, feature contributions should generally
+be created on topic branches branching from the _develop_ branch.
+Submissions would then be made as Pull Requests (PRs) back to the
+_develop_ branch.
+
+## Contributor Setup
+
+If you wish to contribute to the insights-core- project you'll need to
+create a fork in github.  Then
+
+1. Clone your fork::
+
+    git clone git@github.com:your-user/insights-core.git
```

Comment:
A few adjustments from insights-core to insights-core-messaging needed here and below